### DISCUSSION TITLE
配置连接到反向WebSocket Universal服务器 ws://nonebot:8080/onebot/v11/ws/ 时出现错误

### DISCUSSION THREAD

**Discussion by gitmithy** [MountainDash/nonebot-bison#188] [Q&A]
配置连接到反向WebSocket Universal服务器 ws://nonebot:8080/onebot/v11/ws/ 时出现错误: dial tcp: lookup nonebot on 127.0.0.11:53: server misbehaving
请教下这里的nonebot:8080是要改成自己的服务器ip吗，改成localhost是这么报错
连接到反向WebSocket Universal服务器 ws://localhost:8080/onebot/v11/ws/ 时出现错误: dial tcp 127.0.0.1:8080: connect: connection refused

**Comment by felinae98** [comment]
请问你的部署方式？docker还是别的

**Comment by gitmithy** [comment]
> 请问你的部署方式？docker还是别的

是docker

**Comment by gitmithy** [comment]
直接运行部署成功了😂

**Comment by felinae98** [comment]
用的docker-compose吗，如果不是的话需要两个容器运行在相同的网桥中

**Comment by gitmithy** [comment]
> 用的docker-compose吗，如果不是的话需要两个容器运行在相同的网桥中

大佬想请问下custom post怎么启用啊，有没有参考啊，markdown的生成位置，对应那个css模板标签应该怎么对应，纯python小白

**Reply by AzideCupric** [reply]
我可以说custom post还没有人用过吗(

**Reply by AzideCupric** [reply]
使用Custom_post其实类似此处的结构
https://github.com/felinae98/nonebot-bison/blob/01060f8d58ddfc66db49abfa1b6c663c684c3c7a/src/plugins/nonebot_bison/platform/weibo.py#L179-L185

相应的修改为
```python
return CustomPost(message_segments=ms_list)
```
*ms_list 指对应的list[MessageSegment]，详细的介绍参见->[什么是CustomPost](https://nonebot-bison.netlify.app/dev/#%E4%BB%80%E4%B9%88%E6%98%AFnonebot-bison-post-custompost)<-

**Comment by felinae98** [comment]
> 大佬想请问下custom post怎么启用啊，有没有参考啊，markdown的生成位置，对应那个css模板标签应该怎么对应，纯python小白

你是打算要二次开发吗
